services/mac_manager/src/main/java/com/futurewei/alcor/macmanager/entity/OuiState.java

```diff
@@ -0,0 +1,38 @@
+/*Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.macmanager.entity;
+
+import lombok.Data;
+
+import java.io.Serializable;
+
+@Data
+public class OuiState implements Serializable {
```

Comment:
OUI is Orgarnization Unit Identifier.  A mac address is composed of two parts (OUI and NIC).

Reply:
Cool. Let us leave some comments on the source file so that the community could understand as well. Ideally we could add some links to external resources.

Reply:
After our discussion, OuiState is removed and now it is configurable.